Assets/Editor/Tests/CameraTest.cs

```diff
@@ -0,0 +1,42 @@
+using NUnit.Framework;
+using Altom.AltUnityDriver;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+
+public class CameraTest : MonoBehaviour
+{
+    private AltUnityDriver altUnityDriver = new AltUnityDriver();
+    private string SceneName = "Main";
```

Comment:
altUnityDriver -> _altUnityDriver, and can be made readonly
SceneName: naming convention and can be made constant